cypress/plugins/index.js

```diff
@@ -34,11 +35,9 @@ module.exports = (on, config) => {
           postedAt: now,
           lastSubthreadActivity: now,
         }));
-      MongoClient.connect("mongodb://localhost:27017", async function(err, client) {
-        if(err) {
-          console.error(err);
-          return;
-        }
+      const client = new MongoClient(process.env.TESTING_DB_URL);
+      try{
+        await client.connect();
```

Comment:
For a reason I haven't figured out yet, updates stopped working on the remote database if I used the `MongoClient.connect()` syntax. `updateOne` gave a successful output (1 document found, 1 document modified), but that change wouldn't show up in the db and the tests would fail. The problem went away with a switch to this syntax. My best guess is that this had something to do with the db connection from `dropAndSeedDatabase` not closing, though nothing I tried managed to resolve that using the prior syntax, and I'm not totally sure why that failure happened on the server db but not on the in-memory db. Having spent several hours debugging this I'm going to cut my losses on further investigation until/unless it rears its head again.

Reply:
😢

Reply:
Add a comment please.